pom.xml

```diff
@@ -5,9 +5,16 @@
     <modelVersion>4.0.0</modelVersion>
     
     <groupId>com.lightstep.tracer</groupId>
-    <artifactId>java-common</artifactId>
-    <version>0.13.2</version>
-    <packaging>jar</packaging>
+    <artifactId>java</artifactId>
+    <version>0.13.2-SNAPSHOT</version>
```

Comment:
Are we going to move to snapshots? This might mess up the make publish stuff.

Reply:
I think we have to move to snapshot if we want the tests to pass (otherwise maven tries to pull in the cached artifact over compiling the code locally).

We should still publish without the snapshot though. What’s the publish path? Is it in the makefile? I can try and fix it if it’s broken.

Reply:
It's in the makefile, It's pretty delicate.

Reply:
I got around this in the lightstep-tracer-java project by adding `mvn install` to the dependency overrides in the Circle configuration (the website, not the file)

Reply:
Looking at other open source multi-project java projects, they let the pom file stay at SNAPSHOT until they are ready to publish a new version to maven (https://github.com/square/wire/commit/fd0d53076ba6b1896280226f8ba9e470a990b8e6). Given that, I think it makes sense to leave it as SNAPSHOT for now until we are ready to publish the new artifact, does that make sense?

Reply:
Sounds fine, just make sure all the Make scripts and what not are still working.

Reply:
Verified `make build` still works!